mkosi/config.py

```diff
@@ -4111,10 +4122,29 @@ def format_bytes_or_none(num_bytes: Optional[int]) -> str:
     return format_bytes(num_bytes) if num_bytes is not None else "none"
 
 
-def summary(config: Config) -> str:
-    def bold(s: Any) -> str:
-        return f"{Style.bold}{s}{Style.reset}"
+def bold(s: Any) -> str:
+    return f"{Style.bold}{s}{Style.reset}"
+
+
+def cat_config(images: Sequence[Config]) -> str:
+    c = io.StringIO()
+    for n, config in enumerate(images):
+        if n > 0:
+            print(file=c)
+        print(bold(f"### IMAGE: {config.image or 'default'}"), file=c)
+        for path in config.config_paths:
+            # Display the paths as relative to ., if underneath.
+            try:
+                path = path.relative_to(Path.cwd())
+            except ValueError:
+                pass
```

Comment:
I'd just use `path.is_relative_to()` here, it'll be shorter and you get rid of the false positive from CodeQL at the same time. Sure there's the whole ask for forgiveness thing rather than permission but I think we can ignore that here.